plugin/tests/insert_use.vader

```diff
@@ -9,74 +9,13 @@ Given php (source with no namespace):
   }
 Do (put the cursor over an existing class name and add use):
   /Phpactor\<Enter>
-  :call PhactUseAdd()\<Enter>
+  :call phpactor#UseAdd()\<Enter>
 
 Expect php (the use statement to be inserted):
   <?php
-  use Phpactor\Phpactor;
```

Comment:
This test has been here for ages, and never worked before. All of the test cases are now handled in PHP.